### PR TITLE
Deployment flow break, Admin can create cloud for anybody and non authenticated users will see Free trial option

### PR DESCRIPTION
**Pull request by sandesh556**
This PR focuses on breaking the deployment flow for the cloud that is now a client when they click order now from the store they will see a pop-up to create the cloud domain and then it's stored with us and only after payment is done is when the client receives the cloud creation email.

Admin will be able to create a cloud instance for any user when they execute the cloud order.

Also non authenticated users will see the START FREE TRIAL option when they click on it they will be redirected to login page and after they login a pop up will auto appear pre-filled and ready to go

### CODE REVIEW

**Comment by DhanyaChengappa** [comment]
**Demo feedback:**

- [ ] Domain name can be below product name or any other suitable place within the item borders

Screenshot 2023-09-26 at 1 32 44 PM

**Comment by sandesh556** [comment]
Screen Shot 2023-09-26 at 2 42 53 PM

**Comment by sandesh556** [comment]
- Verify Button is removed or not I mean deploy from everywhere.

File: app/Http/Controllers/Tenancy/TenantController.php [hunk 1]
```diff
@@ -197,6 +197,7 @@ public function createTenant(Request $request)
 
         try {
             $company = (string) $request->input('domain');
+            dd($company);
```
Comment (RafficMohammed) [inline app/Http/Controllers/Tenancy/TenantController.php:200]: Remove the dd

File: bootstrap/cache/services.php [hunk 2]
```diff
@@ -38,56 +38,57 @@
         33 => 'GrahamCampbell\\Markdown\\MarkdownServiceProvider',
```
Comment (RafficMohammed) [inline bootstrap/cache/services.php:38]: 🚩. Remove this cache file bro

File: resources/views/themes/default1/front/postPaymentTemplate.blade.php [hunk 3]
```diff
@@ -72,11 +72,18 @@
 
 @if($downloadPermission['downloadPermission'] == 1 && $product->type != '4')
 
+      <a href= product/download/{{$order->product}}/{{$invoice->number}} " class="btn btn-sm btn-primary btn-xs" style="margin-bottom:15px;"><i class="fa fa-download" style="color:white;"> </i>&nbsp;&nbsp;Download the Latest Version here</a>
```
Comment (RafficMohammed) [inline resources/views/themes/default1/front/postPaymentTemplate.blade.php:75]: href ="{{ url("product/download/$order->product/$invoice->number") }}"

**Review by RafficMohammed — CHANGES_REQUESTED** [review]

**Review by SarfrazS — COMMENTED** [review]
Below Features we are added in this pr:-

1. Authenticated user also can see cloud free trial button.
image
2. When authenticated user click on free trial below alert message display.
image
3. Unauthenticated users click on cloud free trial and then click on login, after login cloud creation popup will display.
image

5. Unauthenticated users click on cloud free trial and then click on register and then click on login after login cloud creation popup will display.
image
6. Login and click on cloud free trial that time domain name will filled with your company name.
7. Admin can deploy cloud instance. Create invoice and execute invoice. after that deploy will done for the user. 
image
8. User will receive two mails one for purchase and another for cloud creation.

**Review by SarfrazS — APPROVED** [review]
Above all features are working fine.
**First-Round Test Summary:**
**Total time in days/hours: 5 hours**
**Total issues: 2**
Minor: 2
Major: 0 
Critical: 0

**Pull request merged**